### PR TITLE
CR-1064049 Added an exception instead of just cout message if any of the System operations fails

### PR DESCRIPTION
**Pull request by venkatp-xilinx**
Added an exception instead of just cout message if any of the System operations fails.

Reviewer: @vboggara-xilinx 
CR: 1064049

### CODE REVIEW

File: src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx [hunk 1]
```diff
@@ -3,10 +3,19 @@
...
+        throw status; 
+      } else {
+        return;
+      }
+    } 
+    catch (int status) { 
+      std::cout << "ERROR: [EMU 60-600] " << command.c_str() << " Exception Caught - Failed with the error code " << status << ". PLEASE CHECK YOUR PERMISSIONS " << std::endl;
+    } catch (...) {
+      std::cout << "ERROR: [EMU 60-600] " << command.c_str() << " Exception Caught - Failed with the error code " << status << ". PLEASE CHECK YOUR PERMISSIONS " << std::endl;
+    }
```
Comment (stsoe) [inline src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx:18]: I don''t understand this code at all?   Throwing the status and catching it is a no-op and second catch is never applicable.

File: src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx [hunk 2]
```diff
@@ -3,10 +3,19 @@
...
   }
   
   void makeSystemCall (std::string &operand1, systemOperation operation, std::string operand2 )
```
Comment (stsoe) [inline src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx:21]: I don't see operand1 modified by this function?   And why is operand2 passed by value?

File: src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx (hunk 1 above)
Comment (venkatp-xilinx) [reply src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx]: removed the second catch which never hits.

File: src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx (hunk 2 above)
Comment (venkatp-xilinx) [reply src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx:19]: corrected the operand to pass by reference

File: src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx [hunk 3]
```diff
@@ -2,11 +2,18 @@
...
+    try { 
+      if (status) { 
+        throw status; 
+      } else {
+        return;
+      }
+    } 
+    catch (int status) { 
+      std::cout << "ERROR: [EMU 60-600] " << command.c_str() << " Exception Caught - Failed with the error code " << status << ". PLEASE CHECK YOUR PERMISSIONS " << std::endl;
+    }
```
Comment (stsoe) [inline src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx:16]: Hi @venkatp-xilinx Still makes no sense to me can you explain the reason for this try/catch?  It's a no-op, what's the point throwing and catching immediately?

Comment (venkatp-xilinx) [reply src/runtime_src/core/pcie/emulation/common_em/system_utils.cxx]: Thanks for the suggestion. My requirement is to immediate exit from the flow when any of the system operation fails. Now I have corrected the code accordingly

**Pull request merged**